doc/modules/cross_validation.rst

```diff
@@ -548,6 +582,21 @@ to shuffle the data indices before splitting them. Note that:
 * To ensure results are repeatable (*on the same platform*), use a fixed value
   for ``random_state``.
 
+Cross validation of time series data
```

Comment:
That is a valid point. You could move this section above the note on shuffling and move the HomogeneousCV object to after this, so that it reads like:

PredefinedSplit

Crossvalidation of time series data:
 - HomogeneousTimeSeriesCV

A note on shuffling